### PR TITLE
Find a Rep MVP staging review

### PR DESCRIPTION
**Pull request by cosu419**
## Summary
- Replacing `<strong>` tag with `<h3>` for accessibility
- Search now being handle with enter key
- Addressed sorting bug where error was being thrown on updating repType after updating sort

## Issues
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/71165
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/71434

### CODE REVIEW

**Review by rmessina1010 — DISMISSED** [review]
LGTM

File: src/applications/representative-search/components/search/SearchResultsHeader.jsx [hunk 1]
```diff
@@ -13,11 +13,6 @@ export const SearchResultsHeader = props => {
 
   const noResultsFound = !searchResults || !searchResults.length;
 
-  const sortOptions =
```
Comment (jvcAdHoc) [inline src/applications/representative-search/components/search/SearchResultsHeader.jsx:16]: I think this means we'll always use the org sort options? I'm unable to test locally right now.

File: src/applications/representative-search/config.js [hunk 2]
```diff
@@ -27,23 +27,23 @@ export const orgSortOptions = {
...
-export const individualSortOptions = {
-  distance_asc: 'Distance (closest to farthest)',
-  distance_desc: 'Distance (farthest to closest)',
-  last_name_asc: 'Last Name (A - Z)',
-  last_name_desc: 'Last Name (Z - A)',
-};
+/*
+ * Toggle true for local development
+ */
+export const useStagingDataLocally = true;
```
Comment (jvcAdHoc) [inline src/applications/representative-search/config.js:33]: This got flipped in the latest commit

File: src/applications/representative-search/components/search/SearchResultsHeader.jsx (hunk 1 above)
Comment (jvcAdHoc) [reply src/applications/representative-search/components/search/SearchResultsHeader.jsx:16]: Talked offline and this is intentional for now

**Review by jvcAdHoc — DISMISSED** [review]

**Review by jvcAdHoc — DISMISSED** [review]

**Review by rmessina1010 — DISMISSED** [review]
LGTM

**Review by ataker — DISMISSED** [review]

**Review by ataker — APPROVED** [review]

**Pull request merged**